pkg/universe.dagger.io/go/test.cue

```diff
@@ -2,13 +2,30 @@ package go
 
 // Test a go package
 #Test: {
-	// Package to test
-	package: *"." | string
+	// DEPRECATED: use packages instead
+	package: string | *null
+
+	// Packages to test
+	packages: [...string]
 
 	#Container & {
 		command: {
+			_packages: [...string] | []
+			if package == null && len(packages) == 0 {
+				_packages: ["."]
+			}
+			if package != null && len(packages) == 0 {
+				_packages: [package]
+			}
+			if package == null && len(packages) > 0 {
+				_packages: packages
+			}
+			if package != null && len(packages) > 0 {
+				_packages: [package] + packages
+			}
```

Comment:
Add a `FIXME: find a better workaround with disjunction`

Maybe someone will find it :cry:

Reply:
Also, just if we could factor the one in the build.cue and test.cue, that would be good. I didn't succeed, though.